src/types/CreateTaskOptions.interface.ts

```diff
@@ -10,6 +11,7 @@ export interface CreateBatchExecTaskOptions {
   web3FunctionHash: string;
   web3FunctionArgs?: Web3FunctionUserArgs;
   web3FunctionArgsHex?: string;
+  triggerConfig: TriggerConfig;
 }
 export interface CreateTaskOptions {
```

Comment:
we should also remove the top level
```
  startTime?: number;
  interval?: number;
```
to avoid any confusion for the user and make sure they use the new trigger type

Reply:
I've added back for `dev` UI compatibility to easily upgrade the `automate-sdk` on it easily. Should we completely remove `startTime` and `interval` from SDK?

Reply:
removed in 3b64174c8077471c0cfc82f41542298516312877